com.adilhanney.saber.json

```diff
@@ -0,0 +1,105 @@
+{
+	"app-id": "com.adilhanney.saber",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "22.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "saber",
+	"separate-locales": false,
+	"finish-args": [
+		"--share=network",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland",
+		"--device=dri",
+		"--socket=pulseaudio",
+		"--filesystem=xdg-documents/Saber/:create"
+	],
+	"modules": [
+		{
+			"name": "saber",
+			"buildsystem": "simple",
+			"only-arches": [
+				"x86_64"
+			],
```

Comment:
```suggestion
```

Reply:
@barthalion Is this not needed to be valid JSON?

Reply:
> @barthalion Is this not needed to be valid JSON?

For some reason, Github only coloured in the last line as being removed. Nevermind